src/test/java/string/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package string;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("1,1,2,3을 넣으면 size는 3이다.")
+    void getSize() {
+        assertThat(this.numbers.size()).isEqualTo(3);
```

Comment:
컬렉션 검증용 메서드도 활용해 보시면 좋을 것 같아요
```suggestion
        assertThat(numbers).hasSize(3);
```